Source/PauseMenu/TabView.cs

```diff
@@ -172,6 +172,8 @@ public void ProcessControls()
                     Tabs[Index].Active = false;
                     Tabs[Index].Focused = false;
                     Tabs[Index].Visible = false;
+                    if (Tabs[Index] is TabSubmenuItem)
+                        (Tabs[Index] as TabSubmenuItem).RefreshIndex();
```

Comment:
This can be done with a single type check:
```suggestion
                    if (Tabs[Index] is TabSubmenuItem submenuItem)
                        submenuItem.RefreshIndex();
```